resources/public/api/conf/2.0/schemas/uk_property_non_fhl_retrieve/response.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Retrieve a Historic Non-FHL UK Property Business Annual Submission",
+  "description": "This endpoint allows a developer to retrieve the income from adjustments and allowances for their non-FHL UK property business for tax years up to 2021-22. A National Insurance number and tax year must be provided.",
+  "type": "object",
+  "properties": {
+    "annualAdjustments": {
+      "type": "object",
+      "description": "Object containing the details about annual adjustments.",
+      "properties": {
+        "lossBroughtForward": {
+          "type": "number",
+          "description": "Loss brought forward from earlier years.",
+          "multipleOf": 0.01,
+          "minimum": 0.00,
+          "maximum": 99999999999.99,
+          "example": "2000.99"
+        },
+        "privateUseAdjustment": {
+          "type": "number",
+          "description": "Private use adjustment on an asset by a sole trader or partner in a partnership, the asset is put in to a single-asset pool and allowances are restricted on the business to private-use ratio.",
+          "multipleOf": 0.01,
+          "minimum": 0.00,
+          "maximum": 99999999999.99,
+          "example": "1000.45"
+        },
+        "balancingCharge": {
+          "type": "number",
+          "description": "Balancing charge on sale or cessation of business use (where you have disposed of assets for more than their tax value).",
+          "multipleOf": 0.01,
+          "minimum": 0.00,
+          "maximum": 99999999999.99,
+          "example": "1000.45"
+        },
+        "businessPremisesRenovationAllowanceBalancingCharges": {
+          "type": "number",
+          "description": "Income from the sale or grant of a long lease for a premium of renovated business premises within 7 years of first use.",
+          "multipleOf": 0.01,
+          "minimum": 0.00,
+          "maximum": 99999999999.99,
+          "example": "5000.99"
+        },
+        "nonResidentLandlord": {
```

Comment:
"nonResidentLandlord" and "jointlyLet"(line 52) are currently listed as optional in the RAML but the spec states they are mandatory.
<img width="632" alt="image" src="https://user-images.githubusercontent.com/106095575/180181603-36d228c9-32c0-49b9-ba60-fcd0391efa68.png">
<img width="834" alt="image" src="https://user-images.githubusercontent.com/106095575/180181706-41226698-957a-40f4-b241-7d5fde0cebcd.png">

Reply:
jointlyLet and nonResidentLandlord are now mandatory